main.py

```diff
@@ -200,7 +200,8 @@ def handle_link_up(self, event):
                 log.debug(f'{data.get("id")} can not be provisioning yet.')
                 continue
 
-            evc.handle_link_up(event.content['link'])
+            if evc.schedule_active:
+                evc.handle_link_up(event.content['link'])
```

Comment:
To me, `schedule_active` reads like "link was activated by the scheduler", but it really means "should be active", so I think we need a better name for this flag.